dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -227,13 +227,13 @@ FEATURE_NOT_IMPLEMENTED.useraction=Avoid using the feature, or switch to a drive
 # Using 4000-4999 for Liberty-only messages
 # 
 
-DSRA4000.no.suitable.driver=DSRA4000E: A valid JDBC driver implementation class was not found for the jdbcDriver {0} using the library {1}.
-DSRA4000.no.suitable.driver.explanation=A valid JDBC driver implementation class was not found.
-DSRA4000.no.suitable.driver.useraction=Update the configuration so that either the library contains a path to a recognized JDBC driver or the implementation class is specified on the data source.
+DSRA4000.no.suitable.driver=DSRA4000E: No implementations of {0} are found for {1} with library {2}. Searched in: {3}. Searched under packages: {4}.
+DSRA4000.no.suitable.driver.explanation=Reasons why a data source or driver implementation might not be found include: Name or location of JDBC driver JAR files might be incorrect or inaccessible. JDBC driver might not properly identify its Driver class, or might be lacking one. Liberty's built-in logic to identify data source implementation classes does not apply to all drivers.
+DSRA4000.no.suitable.driver.useraction=Verify that configuration of the JDBC driver and library are correct. If implementation class are still not found, manually specify them in the configuration.
 
-DSRA4001.no.suitable.driver.nested=DSRA4001E: A valid JDBC driver implementation class was not found in the nested library used by jdbcDriver {0}.
-DSRA4001.no.suitable.driver.nested.explanation=A valid JDBC driver implementation class was not found.
-DSRA4001.no.suitable.driver.nested.useraction=Update the configuration so that either the nested library contains a path to a recognized JDBC driver or the implementation class is specified on the data source.
+DSRA4001.no.suitable.driver.nested=DSRA4001E: No implementations of {0} are found for {1}. Searched in: {2}. Searched under packages: {3}.
+DSRA4001.no.suitable.driver.nested.explanation=Reasons why a data source or driver implementation might not be found include: Name or location of JDBC driver JAR files might be incorrect or inaccessible. JDBC driver might not properly identify its Driver class, or might be lacking one. Liberty's built-in logic to identify data source implementation classes does not apply to all drivers.
+DSRA4001.no.suitable.driver.nested.useraction=Verify that configuration of the JDBC driver and library are correct. If implementation class are still not found, manually specify them in the configuration.
```

Comment:
DSRA4001.no.suitable.driver.nested.useraction=Verify that **the configurations** of the JDBC driver and library are correct. If **the implementation classes** are still not found, manually specify them in the configuration.

Reply:
Thanks for catching these typos. I'm fixing it exactly as you suggested.